SYCL/DeviceLib/built-ins/printf.cpp

```diff
@@ -1,5 +1,11 @@
 // UNSUPPORTED: cuda || hip
 // CUDA and HIP don't support printf.
+
+// esimd_emulator : Warning messages from CM_EMU library for emulation
```

Comment:
@kbobrovs do you think these comments need to be guarded with the ESIMD Intel feature?

Reply:
@vzakhari - no, I don't think it needs to, as this is public feature. But all non-ESIMD tests should be guarded with // UNSUPPORTED: esimd_emulator, as this BE can only run ESIMD tests.

Reply:
So, @dongkyunahn-intel, why bother adding // XFAIL: esimd_emulator rather than // UNSUPPORTED: esimd_emulator? Do you expect this test to pass in future?

Reply:
'XFAIL' is already added below - line 7.

Reply:
Please re-read my question. XFAIL should not be there.